### DISCUSSION TITLE
[v3.1.0]如果在柱状图中使用ignore，可以取消绘制间距吗?

### DISCUSSION THREAD

**Discussion by JayJudge** [XCharts-Team/XCharts#245] [Q&A]
![1676530607547](https://user-images.githubusercontent.com/37801711/219290886-b63f11af-ff0e-4df0-80f9-8415e3106b10.jpg)

如图，有三个serie（serie0,serie1,serie2），serie1的第三个数据为0，我想要实现，serie0和serie2在第三个数据中间没有间距，请问可以实现吗？
或者，柱状图的每个数据项目，可以单独调整宽度吗？

**Comment by monitor1394** [comment]
目前是不支持的。后面版本是可以考虑实现ignore时不占位的功能。
